servers/n8n-mcp/src/server.js

```
@@ -0,0 +1,874 @@
+        fs.writeJSONSync(dataFile, data);
+        args.push('--data', dataFile);
+      }
+      
+      // Set execution ID
+      const executionId = uuidv4();
+      
+      // Execute workflow
+      const process = spawn('n8n', args, {
+        cwd: serverConfig.dataDir,
+        env: {
+          ...process.env,
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)